### PR TITLE
Prayer requests anonymity option

### PR DESCRIPTION
**Pull request by mmorelos2**

### CODE REVIEW

File: app/controllers/prayer_requests_controller.rb [hunk 1]
```diff
@@ -16,7 +16,7 @@ def index
   def show
     prayer_request_id = params[:id]
     if !current_user.is_admin? && PrayerRequest.find(prayer_request_id).user_id != current_user.id
-      redirect_to(prayer_requests_path, alert: 'You are not authorized to perform this action. LOL')
```
Comment (jamsterwes) [inline app/controllers/prayer_requests_controller.rb:19]: Thanks for changing this 😭 why did it say this in the first place LMAO

File: app/controllers/prayer_requests_controller.rb [hunk 2]
```diff
@@ -85,15 +85,38 @@ def set_prayer_request
   end
 
   # Only allow a list of trusted parameters through.
-  def prayer_request_params_create
+  def create_prayer_requests_params
```
Comment (jamsterwes) [inline app/controllers/prayer_requests_controller.rb:88]: This naming scheme is much better. Verb + object follows our implicit naming standards.

File: spec/models/prayer_request_spec.rb
Comment (jamsterwes) [inline spec/models/prayer_request_spec.rb:1]: is there a reason the comments were removed?

File: spec/views/prayer_requests/edit.html.erb_spec.rb [hunk 3]
```diff
@@ -1,20 +1,19 @@
 require 'rails_helper'
 
 RSpec.describe('prayer_requests/edit') do
-  before do
-    @user1 = FactoryBot.create(:user, email: 'user1@gmail.com')
-    @admin = FactoryBot.create(:admin_user, email: 'admin@gmail.com')
-  end
+  let!(:user1) { FactoryBot.create(:user, email: 'user1@gmail.com') }
+  let!(:admin) { FactoryBot.create(:admin_user, email: 'admin@gmail.com') }
```
Comment (jamsterwes) [inline spec/views/prayer_requests/edit.html.erb_spec.rb:5]: thanks for fixing the `@` issue with rubocop, let's look into re-enabling that rubocop rule now

**Review by jamsterwes — APPROVED** [review]
lgtm

File: spec/models/prayer_request_spec.rb
Comment (mmorelos2) [reply spec/models/prayer_request_spec.rb:1]: I think they are not necessary since the code is just as easy to read as the comments.

**Pull request merged**